### PR TITLE
Add TextNow recipe

### PR DESCRIPTION
**Pull request by Arthur-Huan**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.

#### Description of Change

Adding the recipe for TextNow.

### CODE REVIEW

File: recipes/textnow/webview.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+  Ferdium.injectCSS(_path.default.join(__dirname, 'service.css'));
+
+  localStorage.setItem(
+    '_cs_desktopNotifsEnabled',
+    JSON.stringify({
+      __t: Date.now(),
+      __v: true,
+    }),
+  );
```
Comment (SpecialAro) [inline recipes/textnow/webview.js:86]: Why is this needed?

File: recipes/textnow/webview.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      if (typeof notification.options.body !== 'string') {
+        notification.options.body =
+          (((notification.options.body || {}).props || {}).content || [])[0] ||
+          '';
+      }
+
+      return notification;
+    });
+  }
+};
```
Comment (SpecialAro) [inline recipes/textnow/webview.js:104]: This code is interesting 🤔

Why do you need it? Did you by any chance noticed that the title or body could be different from string and that would cause an error?

**Review by SpecialAro — CHANGES_REQUESTED** [review]
Hi! Thanks for raising this PR 😁

I've made some comments on your code if you care to answer.

Generally, LGTM but I don't have a way of testing the code because I don't have an account on this service.

**Comment by SpecialAro** [comment]
Hi @Arthur-Huan!

Did you close this PR by accident?

**Comment by Arthur-Huan** [comment]
No, I realized some of the files were the wrong ones. It's kind of a mess on my computer and I must have mixed some files up.
